src/dispatch/case/flows.py

```diff
@@ -127,10 +164,42 @@ def case_new_create_flow(*, case_id: int, organization_slug: OrganizationSlug, d
                 "Case assignee not paged. No relationship between case type and an oncall service."
             )
 
-    # TODO(mvilanova): we send the case created notification
+    conversation_plugin = plugin_service.get_active_instance(
+        db_session=db_session, project_id=case.project.id, plugin_type="conversation"
```

Comment:
That's a bit defensive for my liking; the Case should never be `None.` There are two functions typically. One that creates the case object, e.g. case_service.create(), and then this function runs the flow for new cases. Are there situations I'm unaware of that would pass `None` to this function?

Reply:
That's fair, I am not aware of that situation. This was from something `mypy` highlighted (because `case = get(db_session=db_session, case_id=case_id)` returns `Optional[Case]`. I'm experimenting w/ type checkers and this is an example of that. I suspect it might help us given the size of the codebase, especially new folks. Maybe here we should use a `one_or_raise` type service function? Just brainstorming, off-topic at this point.